### PR TITLE
*/*: [QA] Remove redundant --docdir/--htmldir

### PR DESCRIPTION
**Pull request by SoapGentoo**
Signed-off-by: David Seifert <soap@gentoo.org>

### CODE REVIEW

File: games-simulation/crrcsim/crrcsim-0.9.13.ebuild [hunk 1]
```diff
@@ -35,11 +35,7 @@ src_prepare() {
 }
 
 src_configure() {
-	econf \
-		--datadir="/usr/share" \
-		--datarootdir="${EPREFIX}/usr/share" \
```
Comment (ulm) [inline games-simulation/crrcsim/crrcsim-0.9.13.ebuild]: `econf` doesn't pass `--datarootdir`. Have you checked that this isn't needed?

File: net-analyzer/bmon/bmon-4.0.ebuild [hunk 2]
```diff
@@ -42,7 +42,5 @@ src_prepare() {
 }
 
 src_configure() {
-	econf \
-		CURSES_LIB="$( $(tc-getPKG_CONFIG) --libs ncurses)" \
-		--docdir="/usr/share/doc/${PF}"
+	econf CURSES_LIB="$( $(tc-getPKG_CONFIG) --libs ncurses)"
```
Comment (ulm) [inline net-analyzer/bmon/bmon-4.0.ebuild]:
```suggestion
	econf CURSES_LIB="$("$(tc-getPKG_CONFIG)" --libs ncurses)"
```
The inner command substitution should be quoted as well.

File: net-analyzer/bmon/bmon-999.ebuild [hunk 3]
```diff
@@ -39,7 +39,5 @@ src_prepare() {
 }
 
 src_configure() {
-	econf \
-		CURSES_LIB="$( $(tc-getPKG_CONFIG) --libs ncurses)" \
-		--docdir="/usr/share/doc/${PF}"
+	econf CURSES_LIB="$( $(tc-getPKG_CONFIG) --libs ncurses)"
```
Comment (ulm) [inline net-analyzer/bmon/bmon-999.ebuild]: Ditto.

File: sys-apps/mount-gtk/mount-gtk-1.4.4.ebuild [hunk 4]
```diff
@@ -37,5 +37,5 @@ src_prepare() {
 src_configure() {
 	# acinclude.m4 is broken and environment flags override these:
 	append-cxxflags -fexceptions -frtti -fsigned-char -fno-check-new -pthread
-	econf --docdir=/usr/share/doc/${PF}
+	default
```
Comment (ulm) [inline sys-apps/mount-gtk/mount-gtk-1.4.4.ebuild]:
```suggestion
	econf
```
Probably a matter of taste, but I'd leave the explicit command in place.

File: sys-fs/ntfs3g/ntfs3g-2017.3.23-r1.ebuild [hunk 5]
```diff
@@ -66,7 +66,6 @@ src_configure() {
 	econf \
 		--prefix="${EPREFIX}"/usr \
```
Comment (ulm) [inline sys-fs/ntfs3g/ntfs3g-2017.3.23-r1.ebuild]: `--prefix` is default as well.

File: sys-fs/ntfs3g/ntfs3g-2017.3.23-r2.ebuild [hunk 6]
```diff
@@ -74,7 +74,6 @@ src_configure() {
 	econf \
 		--prefix="${EPREFIX}"/usr \
```
Comment (ulm) [inline sys-fs/ntfs3g/ntfs3g-2017.3.23-r2.ebuild]: Ditto.

File: app-emulation/ganeti-instance-debootstrap/ganeti-instance-debootstrap-0.11-r1.ebuild [hunk 7]
```diff
@@ -21,10 +21,6 @@ RDEPEND="app-arch/dpkg
 	sys-apps/util-linux
 	sys-fs/e2fsprogs"
 
-src_configure() {
-	econf --docdir=/usr/share/doc/${P}
```
[the remark this answers is not in the document]
Comment (SoapGentoo) [reply app-emulation/ganeti-instance-debootstrap/ganeti-instance-debootstrap-0.11-r1.ebuild]: I can check again, but it's more than likely that this is a mistake and not intended behaviour

File: app-editors/vis/vis-0.5.ebuild [hunk 8]
```diff
@@ -43,8 +43,8 @@ src_prepare() {
 
 src_configure() {
 	./configure \
-		--prefix="${EROOT}usr" \
-		--docdir="${EROOT}usr/share/doc/${PF}" \
+		--prefix="${EPREFIX}"/usr \
+		--docdir="${EPREFIX}"/usr/share/doc/${PF} \
```
[the remark this answers is not in the document]
Comment (SoapGentoo) [reply app-editors/vis/vis-0.5.ebuild]: @mgorny pls comment, I think this was likely an oversight in the review

File: games-simulation/crrcsim/crrcsim-0.9.13.ebuild (hunk 1 above)
Comment (SoapGentoo) [reply games-simulation/crrcsim/crrcsim-0.9.13.ebuild]: verified, no change

File: net-analyzer/bmon/bmon-4.0.ebuild (hunk 2 above)
Comment (SoapGentoo) [reply net-analyzer/bmon/bmon-4.0.ebuild]: added

File: sys-fs/ntfs3g/ntfs3g-2017.3.23-r1.ebuild (hunk 5 above)
Comment (SoapGentoo) [reply sys-fs/ntfs3g/ntfs3g-2017.3.23-r1.ebuild]: removed

File: app-emulation/ganeti-instance-debootstrap/ganeti-instance-debootstrap-0.11-r1.ebuild (hunk 7 above)
[the remark this answers is not in the document]
Comment (SoapGentoo) [reply app-emulation/ganeti-instance-debootstrap/ganeti-instance-debootstrap-0.11-r1.ebuild]: Tamiko confirmed this was an oversight

File: app-editors/vis/vis-0.5.ebuild (hunk 8 above)
[the remark this answers is not in the document]
Comment (mgorny) [reply app-editors/vis/vis-0.5.ebuild]: Yes, oversight most likely.

**Pull request closed without merging**